common/lib/xmodule/xmodule/course_module.py

```diff
@@ -303,6 +303,12 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
```

Comment:
I'm wondering if we should just make a Dict setting for social URLs in general as opposed to making this Facebook-specific